src/main/java/com/example/kanbanserv/domain/Project.java

```diff
@@ -1,4 +1,95 @@
 package com.example.kanbanserv.domain;
 
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
 public class Project {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String projectIdentifier;
+    private String projectName;
+    private String description;
+    private Date start_date;
```

Comment:
we generally use camelCase in java instead of snake_case. Any specific reason for using snake_case here?

Reply:
Thanks for pointing it out. Have made the change